lib/matplotlib/pyplot.py

```diff
@@ -1047,13 +1047,10 @@ def subplots(nrows=1, ncols=1, sharex=False, sharey=False, squeeze=True,
 
     Parameters
     ----------
-    nrows : int, optional, default: 1
-        Number of rows of the subplot grid.
+    nrows, ncols : int, optional, default: 1
```

Comment:
The 'optional' is a bit weird as it is not precised for the other kwargs of `subplots`, neither in `pyplot.py` nor in `figure.py`.